src/util-macset.c

```diff
@@ -416,6 +429,35 @@ static int MacSetTest05(void)
     PASS;
 }
 
+static int MacSetTest06(void)
+{
+    MacSet *ms = NULL;
+    SC_ATOMIC_SET(flow_config.memcap, 128);
+
+    ms = MacSetInit(10);
```

Comment:
nit: The "declare here" comment from last PR was to just do `MacSet *ms = MacSetInit(10);` instead of declaration and assignment is two separate lines of code.